src/con_duct/__init__.py

```diff
@@ -0,0 +1 @@
+from ._version import __version__  # noqa
```

Comment:
Why not do `__version__ = importlib.metadata.version("con-duct")` like in `__main__.py`?  Using two different methods for getting the version at runtime seems silly.